pyp2spec/template.spec

```diff
@@ -12,7 +12,12 @@ Source:         {{source}}
 BuildArch:      noarch
 {%- endif %}
 BuildRequires:  python3-devel
-
+{% if additional_build_requires -%}
+# The package is probably archful, adding a dependency on a compiler
```

Comment:
I wonder if this comment is worth it. It would be just another thing that the maintainers need to delete.
Considering the support for manual_builderequires from the config is gone, I'd leave the gcc BuildRequire without a comment. WDYT?